tests/03-basic-ceos/01-two-ceos.robot

```diff
@@ -46,6 +46,11 @@ Ensure IPv6 default route is in the config file
     Log    ${f}
     Should Contain    ${f}    ipv6 route
 
+Ensure MGMT VRF is in the config file
+    ${f} =    OperatingSystem.Get File    ${EXECDIR}/clab-${lab-name}/${node1-name}/flash/startup-config
+    Log    ${f}
+    Should Contain    ${f}    vrf instance MGMT
+
```

Comment:
would it be more correct to check if router accepts those vrf commands by executing a `show` command against it and checking that the output contains the expected vrf?

Reply:
Updated test -- I may need to iterate as I didn't try to test locally.